### ISSUE TITLE
Site issues

### ISSUE THREAD

**Issue by ollehints** [geocollections/sarv-edit#152]
Site form has a number of issues:
- data are updated behind the scenes if coming from project form (via click on site ID on sites table); as project forms sends project=null, the project set via site form will be overridden. Sometimes many update requests are sent, for instance 8 requests at 9:44: https://edit2.geocollections.info/site/5580 
- round lat long numbers from GPS/browser to 6 digits after comma.
- do not reset previously saved lat long values from GPS -- separate issue on that too
- map should show the saved site location and could additionally show the current GPS provided location using a different marker
- update logs on save
- same issue with related data on attachments - we do not want to send update requests to attachment table, if changing the main form. in fact, we should only add attachment data via site for only once (for user convenience) and afterwards edit it via the existing attachment form /attachment/$ID.
- on site form opening it should show top of the page, not scroll down to files section automatically.
